vaadin-date-time-picker-flow/src/main/java/com/vaadin/flow/component/datetimepicker/DateTimePicker.java

```diff
@@ -538,4 +601,45 @@ public void setDatePickerI18n(DatePickerI18n i18n) {
         this.i18n = i18n;
         datePicker.setI18n(i18n);
     }
+
+    /**
+     * Sets whether the date time picker is marked as input required.
+     *
+     * @param required
+     *        the value of the required to be set
+     */
+    public void setRequired(boolean required) {
+        getElement().setProperty("required", required);
+        this.required = required;
+    }
```

Comment:
Good point! Was thinking about the same, however it will lead to inconsistency of the API between the components.
In case you are not using Binder, you will be setting `required` by yourself. In this case it will look like i.e.:
```
datePicker.setRequired(true); (most probably uses this one as it is shorter)
dateTimePicker.setRequiredIndicatorVisible(true);
```
Anyway, removed from this PR.

Reply:
Thanks!